pages/property-filter/common-props.tsx

```diff
@@ -34,7 +25,8 @@ export const columnDefinitions = [
     sortingField: 'state',
     header: 'State',
     type: 'enum',
-    getLabel: getStateLabel,
+    getLabel: (value: any) =>
+      Array.isArray(value) ? value.map(v => getStateLabel(v)).join(', ') : getStateLabel(value, value),
```

Comment:
Yes, that is done for the case when the value is not an array but a string (when the contains / not contains operator is used). In that case, the value can be partial e.g. "Sto" instead of "Stopped" and it should be accepted just fine.